src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useUID } from 'react-uid';
+import styles from './ProgressBar.module.css';
+import Text from '../Text';
+
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Label associated with naming the progress bar.
+   */
+  label: string;
+  /**
+   * Max value to be represented by the progress bar.
+   */
+  max: number;
+  /**
+   * The amount of segments to be represented by the progress bar.
+   * Recommended max number of segments is 10.
+   */
+  segmentCount: number;
+  /**
+   * The value that each segment represents.
+   */
+  segmentValue: number;
+  /**
+   * An optional unit of measurement to describe the progress.
+   */
+  unit?: string;
+};
+
+/**
+ * BETA: This component is still a work in progress and is subject to change.
+ *
+ * `import {ProgressBar} from "@chanzuckerberg/eds";`
+ *
+ * A progress bar component that indicating the completion progress of a task.
```

Comment:
```suggestion
 * A progress bar component that indicates how much progress has been made on a task.
```
The current wording kinda makes it sound like this is a kind of badge that says the task is done as opposed to communicating how close it is to being done.

Reply:
Good suggestion, will change